lib/Tmdb/Model/Watch/Provider.php

```diff
@@ -0,0 +1,154 @@
+<?php
+
```

Comment:
Not sure if there's anything in the way to use strict type hinting within the library by now? Altough not everything is aligned like that, I'd sure like to see that fixed in time.